### PR TITLE
Create How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md

### PR DESCRIPTION
**Pull request by khabdrick**
Draft

### CODE REVIEW

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
+# How to Build a Flask, ReactJS Application and Deploy to Okteto
+## Introduction 
+
+**[Flask](https://flask.palletsprojects.com/en/2.0.x/)** is a great open source web microframework built with Python. It is straightforward and easy to understand.
+Flask is used to build web applications and also APIs. In this article we will use it to build an API that displays date and time.
+
+ReactJS is an open-source  front-end  JavaScript library used to build user interfaces.
```
Comment (Youngestdev) [inline How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:7]: Can you add a link to the ReactJS website?

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+**[Okteto](https://okteto.com/)** is an open-source tool that runs locally to synchronize application code changes to a running pod in a local or remote Kubernetes cluster. With Okteto we are able to see deployment version of our app as we are developing it.
```
Comment (Youngestdev) [inline How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:9]:
```suggestion
**[Okteto](https://okteto.com/)** is an open-source tool that runs locally to synchronize application code changes to a running pod in your Kubernetes cluster. With Okteto we are able to see the deployment version of our app during the development process.
```

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+Okteto has a development environment which we are going to use instead of working locally. Of course we can build this project with our development but we’ll be using Okteto's development environment to build our app which gives us a production-like development environment.
+
+In this article we will build an application that displays the date and time with React and Flask and deployment will be handled with Okteto
```
Comment (Youngestdev) [inline How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:12]: Do you mind building another sample for this draft? A to-do list application or an application with a bit of complexity will be better off than a date and time display application.

**Review by Youngestdev — CHANGES_REQUESTED** [review]
Hi @khabdrick, I have some comments:

- The application sample in this draft is insufficient to demonstrate the prowess of Flask. Please build a better sample for this draft.
- We prefer the use of `you` over `we` in our articles. Do you mind rewriting the draft ?
- In the course of rewriting, I'll like more explanation on the build process.

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md (hunk 3 above)
Comment (khabdrick) [reply How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:12]: I stated the project I will be building in the outline but there were no queries.
But no worries, I'll make the appropriate changes.

Comment (Youngestdev) [reply How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:12]: Hi, I'm sorry I might have missed the changes you made in the outline. Thank you.

**Review by khabdrick — COMMENTED** [review]
I am done rewriting the article @Youngestdev

**Comment by Youngestdev** [comment]
Thanks @khabdrick! I'll send my review after the community call. You should join too - https://lu.ma/qgm1qff3

**Comment by khabdrick** [comment]
Okay no problem

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md [hunk 4]
```diff
@@ -0,0 +1,647 @@
...
+      - ./frontend:/user/src/app
+    depends_on: 
+      - api
+
+```
+Now run `$ docker-compose build` and then `$ docker-compose up`
+
+You'll see that your react application is running on port `localhost:8080`
+
+![](First-image)
```
Comment (Youngestdev) [inline How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:132]: Can you provide an image of your frontend application running at localhost:8080?

Comment (khabdrick) [reply How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md]: it's the default page that comes when you run `$ npx create-react-app`, which will be impossible to get unless I go through the steps from the beginning.

**Comment by Youngestdev** [comment]
@khabdrick, you're supposed to provide an image not delete the instance in the draft.

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md [hunk 5]
```diff
@@ -0,0 +1,647 @@
...
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^16.13.1",
+    "react-dom": "^17.0.2",
+    "react-router": "5.2.0",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
+  ...
+```
```
Comment (Youngestdev) [inline How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:530]: Can you list the packages you used? It's unethical to manually add packages.

Please send a command such as `npm install x y z`

**Review by khabdrick — COMMENTED** [review]
> @khabdrick, you're supposed to provide an image not delete the instance in the draft.

I am getting to that @Youngestdev

**Review by Youngestdev — CHANGES_REQUESTED** [review]
Can you polish up the display when todos are listed? Add some little styles.

Your draft is ready and once these changes I've requested are provided, we can move to the next stage.

**Comment by Youngestdev** [comment]
> https://github.com/okteto/blog-proposals/pull/40#discussion_r675422498

@khabdrick, I don't mind :)

**Comment by khabdrick** [comment]
@Youngestdev I am done with the changes :)

File: How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md (hunk 5 above)
Comment (Youngestdev) [reply How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md:529]: @khabdrick, can you fix this?

Comment (khabdrick) [reply How-to-Build-a-Flask-ReactJS-Application-and-Deploy-to-Okteto.md]: My bad, I thought I already committed the change
I have fixed it now!

**Review by Youngestdev — APPROVED** [review]

**Pull request merged**